runtime/src/accounts_db.rs

```diff
@@ -17593,16 +17593,15 @@ pub mod tests {
                                     account_count += 1;
                                 }
                                 debug!("space: {space}, lamports: {lamports}, alive: {alive}, account_count: {account_count}, append_opposite_alive_account: {append_opposite_alive_account}, append_opposite_zero_lamport_account: {append_opposite_zero_lamport_account}, normal_account_count: {normal_account_count}");
-                                let size = 100000;
```

Comment:
since we are no longer allocating a fixed size append vec, the size depends on how many accounts get written to it.